package.json

```diff
@@ -1,62 +1,40 @@
+  "version": "3.0.0",
+  "author": "Alexander Pope <alexander.pope@nrk.no>, Simen Sægrov <simen.sagrov@nrk.no",
   "dependencies": {},
   "devDependencies": {
-    "babel-plugin-syntax-trailing-function-commas": "6.22.0",
-    "babel-plugin-transform-async-generator-functions": "6.24.1",
-    "babel-plugin-transform-async-to-generator": "6.24.1",
-    "babel-plugin-transform-es2015-arrow-functions": "6.22.0",
-    "babel-plugin-transform-es2015-block-scoped-functions": "6.22.0",
-    "babel-plugin-transform-es2015-block-scoping": "6.26.0",
-    "babel-plugin-transform-es2015-classes": "6.24.1",
-    "babel-plugin-transform-es2015-computed-properties": "6.24.1",
-    "babel-plugin-transform-es2015-destructuring": "6.23.0",
-    "babel-plugin-transform-es2015-duplicate-keys": "6.24.1",
-    "babel-plugin-transform-es2015-for-of": "6.23.0",
-    "babel-plugin-transform-es2015-function-name": "6.24.1",
-    "babel-plugin-transform-es2015-literals": "6.22.0",
-    "babel-plugin-transform-es2015-object-super": "6.24.1",
-    "babel-plugin-transform-es2015-parameters": "6.24.1",
-    "babel-plugin-transform-es2015-shorthand-properties": "6.24.1",
-    "babel-plugin-transform-es2015-spread": "6.22.0",
-    "babel-plugin-transform-es2015-sticky-regex": "6.24.1",
-    "babel-plugin-transform-es2015-template-literals": "6.22.0",
-    "babel-plugin-transform-es2015-unicode-regex": "6.24.1",
-    "babel-plugin-transform-es5-property-mutators": "6.24.1",
-    "babel-plugin-transform-exponentiation-operator": "6.24.1",
-    "babel-plugin-transform-object-rest-spread": "6.26.0",
-    "buddy": "6.x.x",
-    "chai": "^4.1.1",
-    "karma": "^1.7.0",
+    "@types/chai": "^4.1.2",
+    "@types/karma": "^1.7.3",
+    "@types/mocha": "^2.2.48",
+    "chai": "^4.1.2",
+    "karma": "^2.0.0",
     "karma-chai": "^0.1.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-mocha": "^1.3.0",
-    "karma-mocha-reporter": "^2.2.4",
-    "mocha": "*"
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-typescript": "^3.0.12",
+    "mocha": "*",
+    "prettier": "^1.11.1",
+    "ts-node": "^5.0.1",
+    "typescript": "^2.7.2"
   },
-  "main": "index.js",
+  "main": "dist/index",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
   "repository": "https://github.com/YR/classlist.git",
   "license": "MIT",
   "scripts": {
-    "prepublish": "buddy build",
-    "test": "buddy build && karma start --single-run --browsers ChromeHeadless karma.conf.js"
-  },
-  "buddy": {
-    "build": [
-      {
-        "input": "src/",
-        "output": ".",
-        "bundle": false,
-        "version": "es5"
-      },
-      {
-        "input": "test/src/test.js",
-        "output": "test/test.js",
-        "label": "test"
-      }
-    ]
+    "prebuild": "npm run clean",
+    "build": "npm run build-mjs && npm run build-cjs",
+    "build-cjs": "tsc -m commonjs --target ES5",
+    "build-mjs": "tsc && mv dist/index.js dist/index.mjs",
```

Comment:
Can you use `--outFile` instead? Currently the `build-mjs` and `build-cjs` scripts have to be run in exactly that order to prevent the `index.js` file generated by `build-cjs` from being overwritten, which is needlessly confusing for future developers.

Reply:
Agree, but as we discussed. This is the the way we have to do it for the moment. See https://github.com/Microsoft/TypeScript/issues/18442